modules/ROOT/pages/configure-teams.adoc

```diff
@@ -0,0 +1,126 @@
+= Configure Teams
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Every root organization has a main _parent team_ that appears by default. Every user in your organization is a member of the highest parent team and receives all permissions therein.
+You can create _child teams_ under the parent team and configure those teams to have different sets of permissions.
+After you create child teams and assign permissions, you can add users to each of your teams according to what permissions the users should have.
+Further, you can assign _team maintainers_ who can modify users and permissions within the teams they maintain.
+
+Users who have the Organization Administrator permission can also change the name of a team, move child teams under different parent teams, or delete teams.
```

Comment:
- Org Admins of the root org can do this for all teams
- Team maintainers can do this for teams they maintain

Reply:
Hmm... I'm wondering if maybe it'll be easier to phrase the Org Admin / Team Maintainer interactions differently. Being an Org Admin can make you a Team Maintainer, in certain circumstances. 

- All Admins can create teams under the Root Team. They will automatically become maintainers of the teams they create.
- All Admins can assign permissions for their business group to any of existing teams or users
- Admins of the Root Org are Team Maintainers of the Root Team by default
- A Team Maintainer is automatically a maintainer of all child teams of the team they maintain
- And then this statement becomes 
```
Maintainers can do the following operations on their teams: 
- Change their names
- Add or remove members
- Add or remove additional maintainers
- Move their teams under different parent teams (that they also maintain)
- Delete them
```

without having to qualify that maintainers or some types of org admins can do it every time

Reply:
Yeah, I prefer this approach. I think I'll go with it. Thank you for this!!